lib/cisco_node_utils/vlan.rb

```diff
@@ -183,5 +196,21 @@ def interfaces
       end
       interfaces
     end
+
+    def mapped_vni
+      vni = config_get('vlan', 'mapped_vni', vlan: @vlan_id)
+      vni.to_i
+    end
+
```

Comment:
This can be simplified as follows.

**Yaml Entry**
```
mapped_vni:
  /N(3|9)/:
    kind: int  <----------------------- Add this
    config_get: 'show running vlan'
    config_get_token: ['/^vlan <vlan>$/', '/^vn-segment (\d+)$/']
    config_set: ['vlan <vlan>', '<state> vn-segment <vni> ; end']
    default_value: ''
```

```
def mapped_vni
  config_get('vlan', 'mapped_vni', vlan: @vlan_id)
end
```